src/pages/Admin/Room/Edit/RoomEditModal.tsx

```diff
@@ -183,6 +183,24 @@ export const RoomEditModal: React.FC<RoomEditModalProps> = ({
             </div>
           </Form.Row>
 
+          <Form.Row>
+            <div className="room-edit-modal__input">
+              <Form.Label>Change room type</Form.Label>
+              <Form.Control
+                as="select"
+                custom
+                name="type"
+                ref={register}
+                defaultValue={room.type}
+              >
```

Comment:
Why are we using `custom` here? (not saying it's wrong, just curious what it's used/needed for)

It seems like this may relate to the following?

- https://react-bootstrap.github.io/components/forms/#forms-custom-select

Reply:
Looking at https://react-bootstrap.github.io/components/forms/ it sounds like we should potentially be wrapping `Form.Label` and `Form.Control` within a `Form.Group` as well? I know that probably doesn't seem to be the existing pattern here, but how does it look when we do that? If it still looks fine, probably good to do (even if we only add it to this part and consider the others OOS for a future cleanup or similar)

Reply:
> Why are we using `custom` here? (not saying it's wrong, just curious what it's used/needed for)
> 
> It seems like this may relate to the following?
> 
> * https://react-bootstrap.github.io/components/forms/#forms-custom-select

Using the class `custom-select` to customize the look of the select. Styled it to look more like the input with a grey background. I can remove it if it requires design approval.

Reply:
> Form.Group

It looks different, in the attached image the lower one is nested in a Form.Group element. I would prefer to have the same convention across the file and do the markup cleanup in a different exercise. 
![image](https://user-images.githubusercontent.com/2968254/114048541-4c7b2a80-98a8-11eb-8673-c691de5ca2ee.png)

Reply:
> I would prefer to have the same convention across the file and do the markup cleanup in a different exercise.

I'm ok with this, so long as we make sure to do it. Can you please create a backlog issue detailing that change to be made.

Reply:
I am removing the custom styling as it is out of scope for this change.
a682ab592733936af071015f3dae38aaf57cecb3